drivers/power/adp1050/adp1050.c

```diff
@@ -346,49 +353,74 @@ int adp1050_pwm_duty_cycle(struct adp1050_desc *desc, uint16_t pulse_width,
 		redge_msb = no_os_field_get(ADP1050_EDGE_MSB_MASK, pulse_start);
 		lsb = no_os_field_get(ADP1050_FALLING_EDGE_LSB_MASK,
 				      pulse_width + pulse_start) | no_os_field_get(ADP1050_RISING_EDGE_LSB_MASK,
-						      pulse_start);
+					      pulse_start);
 
 		reg_fedge = ADP1050_OUTA_FALLING_EDGE_TIMING;
 		reg_redge = ADP1050_OUTA_RISING_EDGE_TIMING;
 		reg_lsb = ADP1050_OUTA_RISING_FALLING_TIMING_LSB;
-
```

Comment:
remove these kinds of changes

Reply:
dont include these changes: removed/added empty lines

Reply:
copy